.github/workflows/test.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       - uses: cachix/cachix-action@v12
         with:
           name: barretenberg
```

Comment:
Shouldn't this be `Noir` then?

Reply:
this cache name connects barretenberg and noir in one single cache. Cachix documentation does not make it clear if you can have more than one in single workflow so need to research it.